src/components/Minecraft/MinecraftCreateForm.tsx

```diff
@@ -95,37 +181,59 @@ export default function CreateMinecraftInstance({
   }
 
   return (
-    <div className="flex w-[500px] flex-col items-stretch justify-center gap-12 rounded-2xl bg-gray-800 px-12 py-24">
-      <Formik
-        initialValues={initialValues}
-        validationSchema={currentValidationSchema}
-        onSubmit={_handleSubmit}
-        innerRef={formikRef}
-        validateOnBlur={false}
-        validateOnChange={false}
-      >
-        {({ isSubmitting }) => (
-          <Form
-            id={formId}
-            className="flex flex-col items-stretch gap-6 text-center"
-          >
-            {_renderStepContent(activeStep)}
-
-            <div className="flex flex-row justify-between">
-              {activeStep !== 0 ? (
-                <Button onClick={_handleBack} label="Back" />
-              ) : (
-                <div></div>
+    <Formik
+      initialValues={initialValue}
+      validationSchema={currentValidationSchema}
+      onSubmit={_handleSubmit}
+      innerRef={formikRef}
+      validateOnBlur={false}
+      validateOnChange={false}
+    >
+      {({ isSubmitting }) => (
+        <Form
+          id={formId}
+          className="flex h-[560px] w-[812px] items-stretch rounded-2xl bg-gray-850"
+        >
+          <div className="flex w-[180px] border-r border-gray-700 ">
+            <div className="mt-9">
+              {steps.map((section, i) => (
+                <div key={i} className="flex items-center">
+                  <div
+                    className={clsx(
+                      'px-4 py-2 text-left font-sans text-medium font-medium leading-5 tracking-medium text-white/50 ',
+                      activeStep === i && 'font-extrabold text-white'
+                    )}
+                  >
+                    {section}
+                  </div>
+                </div>
+              ))}
+            </div>
+          </div>
+          <div className="relative flex h-full w-[632px] grow flex-col items-stretch overflow-auto p-9 text-center">
+            <div className="relative">
+              {_renderStepContent(
```

Comment:
Some minor scrolling issues with dropdown. It's getting "trapped" inside the big box and making it scroll.
![image](https://user-images.githubusercontent.com/44907675/222987780-5f0d2bf2-aa77-4662-bdff-a7937db018e5.png)